src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -1257,7 +1290,8 @@ def _skip_minimization(testcase, message, crash_result=None, command=None):
   testcase.minimized_keys = testcase.fuzzed_keys
 
   if crash_result:
-    _update_crash_result(testcase, crash_result, command)
+    crash_result_json = _jsonify_crash_result(crash_result, command)
+    _update_crash_result(testcase, crash_result_json)
```

Comment:
Can you add a TODO to make this function do more in postprocess instead of main?